src/SFA.DAS.PR.Data.UnitTests/Setup/ProviderTestData.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.PR.Domain.Entities;
+
+namespace SFA.DAS.PR.Data.UnitTests.Setup;
+public class ProviderTestData
+{
+    public static Provider CreateProvider()
+    {
+        return new Provider()
+        {
+            Ukprn = 12345678,
+            Name = "ProviderName",
+            Created = DateTime.UtcNow,
+            Updated = DateTime.UtcNow
```

Comment:
Around various of the other test data we've started to use Fixtures to create the data with the SFA.DAS.Testing.AutoFixture library, you'll get an example in PermissionRequestsTestData. Reduces the need for boiler plate code to remove settings like ThrowingRecursionBehavior on setup.